wazo_call_logd/cel_interpretor.py

```diff
@@ -292,6 +294,64 @@ def interpret_wazo_user_missed_call(self, cel, call):
         call.source_line_identity = identity_from_channel(cel.channame)
         return call
 
+    def interpret_wazo_call_log_destination(self, cel, call):
+        extra = extract_cel_extra(cel.extra)
+        if not extra:
+            return
+
+        extra_tokens = extra['extra'].split(',')
+        extra_dict = dict()
+        for token in extra_tokens:
+            key = token.split(': ')[0]
+            value = token.split(': ')[1]
+            extra_dict[key] = value
```

Comment:
Ah yes, very nice :)

There's a corner case if `type` is not defined, the code would crash with a Traceback. Could you issue an error and `return` in that case?

Reply:
Also, could you `.strip()` the parsed key and value, in order to avoid errors from extra spaces?

Reply:
Done.